tools/cmake/macros_public.cmake

```diff
@@ -612,7 +612,7 @@ function(ua_generate_nodeset_and_datatypes)
             NAME "types_${UA_GEN_NAME}"
             TARGET_PREFIX "${UA_GEN_TARGET_PREFIX}"
             TARGET_SUFFIX "types-${UA_GEN_NAME}"
-            NAMESPACE_MAP "${NAMESPACE_MAP_DEPENDS}"
+            # NAMESPACE_MAP "${NAMESPACE_MAP_DEPENDS}"
```

Comment:
Remove the argument instead of commenting it out.

Reply:
Left over, removed.